.github/.github/pull_request_template.md

```diff
@@ -0,0 +1,54 @@
+# 📄 PR Description
+
+[Add a description for this pull request. Follow the checklist below.] 
+<!--- Describe your changes in detail including a relevant motivation and context for this change -->
+
+
+# ✒️ Checkliste PR-Author
+
+## Before submitting the PR, please make sure you do the following
+
+### Prepare Code for PR
+* [ ] The automated tests were executed locally (`mvn clean test`), and it was ensured that they passed.
+* [ ] All commented-out code and debug statements have been removed.
+* [ ] A test is included that fails without this PR but passes with it.
+* [ ] The relevant documentation, including code comments and public documentation, has been updated.
+    * [ ] [Qytera Github-Page (github.io)](https://github.com/Qytera-Gmbh/Qytera-Gmbh.github.io)
+
+
+### PR Description
+* [ ] A message body is added that provides context and guidance so that the reviewer can quickly understand what the PR does. In the PR body, this includes. In the PR body, includes:
+    * [ ] The purpose of the pull request, clearly illustrating what problem this PR solves and why this solution is appropriate.
+    * [ ] An overview of the changes, ideally with 1-2 examples that illustrate the changes.
+    * [ ] Links to any additional context such as tracking issues or tickets.
+
+### Review Preparation
+* [ ] Review from appropriate team members is requested.
+- Be prepared to explain and discuss changes.
+
+# 🔍 Checkliste PR-Reviewer
+
+## Before accepting the PR, please make sure you do the following
+
+### Check Preconditions
+* [ ] It is confirmed that the author has fulfilled all preconditions. (The review can only take place once this has been fulfilled.)
+
+### Understand the Context
+* [ ] The PR description is read thoroughly to understand the changes.
+* [ ] It is verified that the changes fulfill the purpose described in the PR.
+* [ ] If the PR belongs to an issue or is intended to represent a user story, it is verified that the requirements and acceptance criteria are met. (If necessary, the author is asked where these were specified.)
+- If things are unclear, please ask the author to add or adjust them in the description or as a comment. 
+
+### Code Quality, Style & Functionality
+* [ ] It is ensured that the code adheres to the projects' [coding standards and style guides](https://qytera.atlassian.net/wiki/spaces/QO/pages/3800727592/QTAF+-+Coding+Standards).
+* [ ] The code is clean, readable and well-organized code. 
+* [ ] The commits are named according to [the team conventions](https://qytera.atlassian.net/wiki/spaces/WIS/pages/3861348360/Commit+und+PR+Konventionen+f+r+QTAF). (If not, the author is informed so that he/she can improve for the next time. There are no changes to the commit history required.)
```

Comment:
This is an internal link whose contentshould probably be moved to a [`CONTRIBUTING.md`](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors) of some sort. But since nobody but Qytera is developing QTAF anyways, I think this is fine for now.

Reply:
Yes, that is correct. I also think that it will work for now.